### PR TITLE
Modify quant-dequant node api to take module object and method name

### PR DESCRIPTION
**Pull request by nishantpdce**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#21407 Modify quant-dequant node api to take module object and method name**&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D15651624/)

Modify api takes module object and method whose graph is
instrumented to insert the quant dequant nodes

Differential Revision: [D15651624](https://our.internmc.facebook.com/intern/diff/D15651624/)

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -1309,7 +1309,8 @@ def forward(self, x):
         torch._C._jit_pass_constant_propagation(scriptM.graph)
         # TODO: Build the qparam_dict from parse_ir directly for this pass
         qparam_dict = _helper_generate_qparam(scriptM, input_data)
-        torch._C._jit_pass_insert_quantdequant(scriptM.graph, qparam_dict)
+        torch._C._jit_pass_insert_quantdequant(scriptM._c, "forward",
```
Comment (jerryzh168) [inline test/test_jit.py]: is this a public API? it starts with underscore, does this mean it might be changed in the future?

Comment (jerryzh168) [reply test/test_jit.py]: could you add a comment here if they are expected to change in the future and what should be the correct API that we want to use?

**Review by jerryzh168 — APPROVED** [review]
added comments about scriptM._c API, the rest of the code lgtm

File: test/test_jit.py (hunk 1 above)
Comment (nishantpdce) [reply test/test_jit.py]: ._c is used to access the moduleObj for the c++ native code. This is how its exposed by JIT as i spoke to them last time.

**Pull request closed without merging**